sea-orm-macros/Cargo.toml

```diff
@@ -23,6 +23,7 @@ syn = { version = "2", default-features = false, features = ["parsing", "proc-ma
 quote = { version = "1", default-features = false }
 heck = { version = "0.4", default-features = false }
 proc-macro2 = { version = "1", default-features = false }
+proc-macro-crate = { version = "3.2.0" }
```

Comment:
can we make this optional, only enable with `seaography` feature?

Reply:
Done